Arrowgene.Ddon.Server/LoginServerSetting.cs

```diff
@@ -8,7 +8,10 @@ public class LoginServerSetting
         [DataMember(Order = 1)] public ServerSetting ServerSetting { get; set; }
         [DataMember(Order = 90)] public bool AccountRequired { get; set; }
         [DataMember(Order = 105)] public uint NoOperationTimeOutTime { get; set; }
-        
+        [DataMember(Order = 110)] public bool KickOnMultipleLogin { get; set; }
+        [DataMember(Order = 111)] public int KickOnMultipleLoginTries { get; set; }
```

Comment:
Should we make these nullable like I did with the modifier settings (so we can assign defaults)? The way out settings file works, it only generates new fields if the file doesn't exist. If it exists and the field is not present, it doesn't assign the default values we have in this file. Instead, it assigns the type default values (like 0 and false).